pubspec.yaml

```diff
@@ -29,11 +29,8 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-
-
-  # The following adds the Cupertino Icons font to your application.
-  # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^1.0.2
+  intl: ^0.17.0
```

Comment:
Doesn't look like we're actually using this dependency right now. What's the future use? DateFormat?